src/components/ProvisionSmartWalletDialog.tsx

```diff
@@ -0,0 +1,115 @@
+import { Dialog, Transition } from '@headlessui/react';
+import clsx from 'clsx';
+import { Fragment } from 'react';
+import { chainConnectionAtom, provisionToastIdAtom } from 'store/app';
+import { useAtomValue, useSetAtom } from 'jotai';
+import { provisionSmartWallet } from 'services/wallet';
+
+// Increment every time the current terms change.
+export const currentTermsIndex = 1;
+
+export type SwingsetParams = { foo: string };
+
+const feeDenom = 10n ** 6n;
```

Comment:
Can't we get this from vstorage?

Reply:
I think it should be available under `published.agoricNames.vbankAsset` although the wallet-app has the same logic as here. My assumption is that changing the actual denom would be a significant change that would allow for some lead time to update, but might be good to confirm that.